src/poprox_recommender/training/train.py

```diff
@@ -168,7 +165,19 @@ def save_model(model, output_dir):
 
     with Task("train NRMS") as task:
         task.save_to_file(args.output_dir / "task.json")
-        train(device, args)
+        train_func = train(device, args)
```

Comment:
If we manually enable the cluster with LensKit's [`init_cluster`](https://lkpy.lenskit.org/latest/api/lenskit.parallel.ray.html#lenskit.parallel.ray.init_cluster), using `init_cluster(global_logging=True)`, it will wire up the subprocess logging elements. Ideal place for this would be after `lenskit.configure` and before the `with` block.